cli/tools/upgrade.rs

```diff
@@ -423,6 +424,7 @@ pub async fn upgrade(
     }
   }
 
+  drop(temp_dir); // delete the temp dir
```

Comment:
redundant?

Reply:
Yes, but purposefully in order to ensure a compile error would occur if the scope of temp_dir accidentally changed